src/containers/Authentication/Authentication.tsx

```diff
@@ -24,7 +24,7 @@ function Authentication({closable = false}: AuthenticationProps) {
     const history = useHistory();
     const location = useLocation();
 
-    const [authenticate, {error, isLoading}] = authenticationApi.useAuthenticateMutation(undefined);
+    const [authenticate, {isLoading}] = authenticationApi.useAuthenticateMutation(undefined);
```

Comment:
Why didn't it work as expected?
I mean isn't auth mutation supposed to return error on error?

Reply:
Mutation result is always `{data, error}`, it cannot be processed by catch without `.unwrap()`. In previous version code in `.then` always ran, that caused the bug with error being ignored. I could use only `.then` as well - `.then(({data, error}) -> {})`, but  form with explicit `.catch` looks better for me